### ISSUE TITLE
Fatal crash when running bot, conflict with get accounts operation

### ISSUE THREAD

**Issue by thrize** [Steem-FOSSbot/steem-fossbot-voter#138]
Observed fatal show stopping bot crash when running normally with the following log

```
Unhandled rejection Error: Callback for function steem_getAccounts_wrapper called twice. Wait.for already resumed the execution.
     at resumeCallback (/app/node_modules/wait.for/waitfor.js:28:40)
     at /app/lib.js:1340:5
     at /app/node_modules/steem/lib/api/transports/http.js:69:9
     at tryCatcher (/app/node_modules/bluebird/js/release/util.js:16:23)
     at Promise._settlePromiseFromHandler (/app/node_modules/bluebird/js/release/promise.js:512:31)
     at Promise._settlePromise (/app/node_modules/bluebird/js/release/promise.js:569:18)
     at Promise._settlePromise0 (/app/node_modules/bluebird/js/release/promise.js:614:10)
     at Promise._settlePromises (/app/node_modules/bluebird/js/release/promise.js:689:18)
     at Async._drainQueue (/app/node_modules/bluebird/js/release/async.js:133:16)
     at Async._drainQueues (/app/node_modules/bluebird/js/release/async.js:143:10)
     at Immediate.Async.drainQueues (/app/node_modules/bluebird/js/release/async.js:17:14)
     at runCallback (timers.js:794:20)
     at tryOnImmediate (timers.js:752:5)
     at processImmediate [as _immediateCallback] (timers.js:729:5)
```

Issue with double call to get accounts wrapper using wait.for, investigate and fix

**Comment by thrize** [comment]
Callback is being called twice, likely due to Steem error on get account function call. Make sure only gets called once regardless of Steem API behavior

**Comment by thrize** [comment]
Looks like it's fixed, keep an eye on this.

**Comment by thrize** [comment]
Available in v0.3.2
